packages/dcos-checks/extra/dcos-checks.timer

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=DC/OS Checks Timer: timer for DC/OS Checks service
+[Timer]
+Unit=dcos-checks.service
```

Comment:
This is redundant as long as the service and timer have the same name. `foo.timer` will default to activating `foo.service`.